### DISCUSSION TITLE
Missing table error

### DISCUSSION THREAD

**Discussion by TESTVNULL** [pawelsalawa/sqlitestudio#3776] [Q&A]
### Details
The SQLite studio is connected with my Netbeans programme, but it pops up a "[SQLITE_ERROR] SQL error or missing database (no such table: ﻿Model)". I am sure I have created a table. But I don't know why it can't open it in the programme. 

### Steps to reproduce
[SQLITE_ERROR] SQL error or missing database (no such table: ﻿Model)

### Operating system
mac os 10.15.4

### SQLiteStudio version
v3.2.1

**Comment by pawelsalawa** [comment]
Does error appear in SQLiteStudio or Netbeans?

I won't be able to help you with Netbeans. I don't know why it throws errors.

**Comment by TESTVNULL** [comment]
I am not sure. I connect SQLite studio as a database to my Netbeans. It connected, but it can't read the table.
<img width="704" alt="Screen Shot 2020-05-25 at 8 08 09 PM" src="https://user-images.githubusercontent.com/63830019/82792580-b58dc580-9ec3-11ea-8d17-49b4f30b62ac.png">
<img width="955" alt="Screen Shot 2020-05-25 at 8 08 38 PM" src="https://user-images.githubusercontent.com/63830019/82792588-b7f01f80-9ec3-11ea-974b-ba5af3011e2f.png">

**Comment by pawelsalawa** [comment]
Hold your mouse over the database (in the left-side panel), a popup will appear, where you will see path to the file. Please confirm it is the same path as you have in netbeans.

**Comment by TESTVNULL** [comment]
<img width="760" alt="Screen Shot 2020-05-25 at 8 17 33 PM" src="https://user-images.githubusercontent.com/63830019/82793356-ddc9f400-9ec4-11ea-97ca-99d4e4c3b4f5.png">

**Comment by pawelsalawa** [comment]
Again, I'm not 100% sure of the cause, because netbeans is not my expertise, although I'm suspecting some building job in netbeans, that erases the database file and creates new one.

I think it's this way, because I see you have this message in SQLiteStudio that the window could not be restored, because database could not be resolved.

Why the database could not be resolved? Was it deleted & recreated by netbeans?

**Comment by TESTVNULL** [comment]
it was deleted then I create a new database

**Comment by pawelsalawa** [comment]
Okay then. In SQLiteStudio everything seems to be fine. I don't know why your netbeans complains.

**Comment by TESTVNULL** [comment]
Same. Everything is working but I don't know why it can't read the table. This is why I am asking for help. Thank you for responding to me.

**Comment by TESTVNULL** [comment]
so do I need to contact with Netbeans?

**Comment by pawelsalawa** [comment]
You can, but I'd recommend you to try to figure it out. Go to that directory within your terminal command line, see if that file is not deleted & recreated. Close SQLiteStudio & reopen it, see if that database is still visible to SQLiteStudio and if has this table. Check if table name does not end with extra whitespace that you didn't see, etc.

Try to nail it down.

**Comment by TESTVNULL** [comment]
I already try those actions that you mention for a few days. But it just doesn't work.

**Comment by pawelsalawa** [comment]
Yeah, I'm helpless then.
